from .base import ApplicationException


class UserAlreadyExistsException(ApplicationException):
    """User already exists exception"""
